index.bs

```diff
@@ -1617,8 +1617,21 @@ that are returned to the caller when a new credential is created, or a new asser
 
     dictionary AuthenticatorAttestationResponseJSON {
         required Base64URLString clientDataJSON;
-        required Base64URLString attestationObject;
+        required Base64URLString authenticatorData;
+        required long long publicKeyAlgorithm;
         required sequence<DOMString> transports;
+
+        // This field is missing if pubKeyCredParams was used to negotiate
+        // a public-key algorithm that the user agent doesn't understand.
+        // (See section “Easily accessing credential data” for a list of
+        // which algorithms user agents must support.) If using such an
+        // algorithm then the public key must be parsed from
+        // attestationObject directly.
+        Base64URLString publicKey;
+
+        // This value contains copies of some of the fields above. See
+        // section “Easily accessing credential data”.
+        required Base64URLString attestationObject;
```

Comment:
I put at the bottom since most RPs shouldn't need it—with the new fields, it's only needed if doing attestation. I hoped that putting it at the bottom would lead most implementators to get the needed information from the easy fields, while obviously keeping the full attestation object for those that need it.

Reply:
I understand better your intent in moving `attestationObject` down, and it makes sense with how much we've diminished the importance of attestation for consumer use cases that I'm okay with this now. I'm going to mark this resolved.